test/integration/test_cli_runners.py

```diff
@@ -42,8 +45,10 @@ def start_ssh_docker(container_name, jobs_directory, port=10022, image='agaveapi
                           'nofile=2048:2048',
                           image]
     subprocess.check_call(START_SLURM_DOCKER)
+    if 'openpbs' in image:
+        time.sleep(PBS_STARTUP_DELAY)
     if sys.platform != 'darwin':
-        subprocess.check_call(['docker', 'exec', container_name, 'usermod', '-u', str(os.getuid()), 'testuser'])
+        subprocess.run(['docker', 'exec', container_name, 'usermod', '-u', str(os.getuid()), 'testuser'])
```

Comment:
we're syncing the userid inside of the container with the userid outside, so permissions of files created in the container match with the host. The userid may or may not exist inside the container, and here it's failing because it already exists: https://github.com/galaxyproject/galaxy/runs/4954149417?check_suite_focus=true#step:14:1141

Reply:
Should we just drop lines 50-51 then (assuming tests pass without it)?